x-pack/plugins/cross_cluster_replication/public/app/store/actions/follower_index.js

```diff
@@ -62,3 +66,164 @@ export const saveFollowerIndex = (name, followerIndex) => (
+          })
+          : i18n.translate('xpack.crossClusterReplication.followerIndex.pauseAction.successSingleNotificationTitle', {
+            defaultMessage: `Follower index '{name}' was paused`,
+            values: { name: response.itemsPaused[0] },
+          });
+
+        toastNotifications.addSuccess(successMessage);
+
+        // If we've just paused a follower index we were looking at, we need to close the panel.
+        // TODO: This is temporary because ES currently removes paused followers from the list.
+        // Remove once issue is fixed: https://github.com/elastic/elasticsearch/issues/37127
+        const followerIndexId = getSelectedFollowerIndexId('detail')(getState());
+        if (response.itemsPaused.includes(followerIndexId)) {
+          dispatch(selectDetailFollowerIndex(null));
+        }
+
+        // Refresh list
+        dispatch(loadFollowerIndices(true));
+      }
+    }
+  })
+);
+
+export const resumeFollowerIndex = (id) => (
+  sendApiRequest({
+    label: t.FOLLOWER_INDEX_RESUME,
+    status: API_STATUS.SAVING,
+    scope,
+    handler: async () => (
+      resumeFollowerIndexRequest(id)
+    ),
+    onSuccess(response, dispatch) {
+      /**
+       * We can have 1 or more follower index resume operation
+       * that can fail or succeed. We will show 1 toast notification for each.
+       */
+      if (response.errors.length) {
+        const hasMultipleErrors = response.errors.length > 1;
+        const errorMessage = hasMultipleErrors
+          ? i18n.translate('xpack.crossClusterReplication.followerIndex.resumeAction.errorMultipleNotificationTitle', {
+            defaultMessage: `Error resuming {count} follower indices`,
+            values: { count: response.errors.length },
+          })
+          : i18n.translate('xpack.crossClusterReplication.followerIndex.resumeAction.errorSingleNotificationTitle', {
+            defaultMessage: `Error resuming follower index '{name}'`,
+            values: { name: response.errors[0].id },
+          });
+
+        toastNotifications.addDanger(errorMessage);
+      }
+
+      if (response.itemsResumed.length) {
+        const hasMultipleDelete = response.itemsResumed.length > 1;
+
+        const successMessage = hasMultipleDelete
+          ? i18n.translate('xpack.crossClusterReplication.followerIndex.resumeAction.successMultipleNotificationTitle', {
+            defaultMessage: `{count} follower indices were resumed`,
+            values: { count: response.itemsResumed.length },
+          })
+          : i18n.translate('xpack.crossClusterReplication.followerIndex.resumeAction.successSingleNotificationTitle', {
+            defaultMessage: `Follower index '{name}' was resumed`,
+            values: { name: response.itemsResumed[0] },
+          });
+
+        toastNotifications.addSuccess(successMessage);
+      }
+
+      // Refresh list
+      dispatch(loadFollowerIndices(true));
+    }
+  })
+);
+
+export const unfollowFollowerIndex = (id) => (
```

Comment:
Nit: The naming is a bit strange to me... but we know how naming can be 😊 I'd like better `unfollowLeaderIndex`

Reply:
I like this suggestion, updating all refs